src/arch/x86_64.rs

```diff
@@ -130,18 +156,14 @@ pub extern "C" fn _start(info: &'static bootinfo::BootInfo) -> ! {
 #[cold]
 pub fn oops(cause: &dyn core::fmt::Display) -> ! {
     use core::fmt::Write;
+    // Safety: we are oopsing. Don't worry about it.
+    // We may unlock writers twice, if we oopsed from an ISR, but...who cares.
+    unsafe { force_unlock_writers() }
 
+    tracing::error!(%cause, "oopsing");
     unsafe { asm!("cli" :::: "volatile") }
```

Comment:
Should we `cli` before we unlock the writers, to avoid an external interrupt after we unlock?